src/jaxsim/api/model.py

```diff
@@ -210,15 +234,44 @@ def build(
             terrain=terrain, parameters=None
         )
 
+        # Build the integrator if not provided.
+        match integrator:
+
+            # If None, build a default integrator.
+            case None:
+
+                integrator = jaxsim.integrators.fixed_step.Heun2SO3.build(
+                    dynamics=js.ode.wrap_system_dynamics_for_integration(
+                        system_dynamics=js.ode.system_dynamics
+                    )
+                )
+
+            # If an integrator class is passed, assume that it is a JaxSim integrator
+            # and build it with the default system dynamics.
+            case _ if isinstance(
+                jaxsim.integrators.Integrator, type(jaxsim.integrators.Integrator)
+            ):
```

Comment:
This will always evaluate to True, skipping the default case. What about this:
```suggestion
            case _ if issubclass(
                integrator, jaxsim.integrators.Integrator
            ):
```

Reply:
Done in https://github.com/ami-iit/jaxsim/pull/252/commits/72346140876228b35c0cfd665367756fbb550f53